client/src/modules/dashboard/Venues/components/VenueForm.tsx

```diff
@@ -175,19 +175,21 @@ const VenueForm: React.FC<VenueFormProps> = (props) => {
           </Select>
         </FormControl>
       )}
-      {fields.map(({ key, isRequired, label, type, step, max, min }) => (
-        <Input
-          key={key}
-          label={label}
-          {...register(key)}
-          type={type}
-          isRequired={isRequired}
-          step={step}
-          max={max}
-          min={min}
-          isDisabled={loading}
-        />
-      ))}
+      {fields.map(({ key, isRequired, label, type, step, max, min }) => {
+        return (
+          <Input
+            key={key}
+            label={label}
+            {...register(key)}
+            type={type}
+            isRequired={isRequired}
+            step={step}
+            max={max}
+            min={min}
+            isDisabled={loading}
+          />
+        );
+      })}
```

Comment:
Just a nitpick, could you revert change in this file? The other changes that were made alongside in here were reverted.

Reply:
Done